src/main/webapp/app/shared/icon-card/icon-card.component.ts

```
@@ -0,0 +1,20 @@
+    selector: 'jhi-icon-card',
+    templateUrl: './icon-card.component.html',
+    styleUrl: './icon-card.component.scss',
+    standalone: true,
+    imports: [ArtemisSharedCommonModule, ArtemisSharedPipesModule],
+})
+export class IconCardComponent {
+    headerIcon = input<IconDefinition>(faCircleInfo);
+
+    headline = input<string>('Title');
+
+    constructor() {}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)